<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Add Item</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="7" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btn_clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QDateEdit" name="cal_manufacture_date"/>
       </item>
       <item row="0" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Item Title</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Date added</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QDateEdit" name="cal_date_added"/>
       </item>
       <item row="1" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Item Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QComboBox" name="cmb_item_type">
         <item>
          <property name="text">
           <string>Computer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Video Player</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QPushButton" name="btn_add">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Date of Manufacture</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="txt_title"/>
       </item>
       <item row="4" column="5">
        <widget class="QLineEdit" name="txt_description"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Item View</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tbl_items">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QPushButton" name="btn_delete">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_title</tabstop>
  <tabstop>cmb_item_type</tabstop>
  <tabstop>cal_date_added</tabstop>
  <tabstop>cal_manufacture_date</tabstop>
  <tabstop>txt_description</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>tbl_items</tabstop>
  <tabstop>btn_delete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
